pkg/trace/sampler/scoresampler_test.go

```diff
@@ -134,7 +135,7 @@ func BenchmarkSampler(b *testing.B) {
 
 	for i := 0; i < b.N; i++ {
 		trace := pb.Trace{
-			&pb.Span{TraceID: 1, SpanID: 1, ParentID: 0, Start: 42, Duration: 1000000000, Service: "mcnulty", Type: "web", Resource: string(rand.Intn(signatureCount))},
+			&pb.Span{TraceID: 1, SpanID: 1, ParentID: 0, Start: 42, Duration: 1000000000, Service: "mcnulty", Type: "web", Resource: fmt.Sprint(rand.Intn(signatureCount))},
```

Comment:
Why did this change have to happen? Just curious...

Reply:
I didn't look into why this didn't cause build failures before, but I think it's a mistake regardless. `string(int)` doesn't do what it looks like it does. See: https://play.golang.org/p/ID0MTA0D9_u

Reply:
Hah, indeed.